prog/vnet/update_firewall_rules.rb

```diff
@@ -88,9 +88,9 @@ class Prog::Vnet::UpdateFirewallRules < Prog::Base
     ip saddr @private_ipv4_cidrs ct state established,related,new counter accept
     ip daddr @private_ipv4_cidrs ct state established,related counter accept
     ip6 saddr @private_ipv6_cidrs ct state established,related,new counter accept
-    ip6 daddr @private_ipv6_cidrs ct state established,related counter accept
```

Comment:
typo in commit message

When we started implementing firewalls,
we decided that the traffic in private network would be **excempt**